packages/mantine/src/components/table/Table.tsx

```diff
@@ -224,6 +229,7 @@ export const Table: TableType = <T,>({
             <Fragment key={row.id}>
                 <tr
                     onClick={() => toggleRowSelection(row)}
+                    onDoubleClick={() => doubleClickAction(row.original)}
```

Comment:
If the doubleClickAction prop is undefined this will throw an error

```suggestion
                    onDoubleClick={() => doubleClickAction?.(row.original)}
```

This makes me realize that you forget to write a unit test for this new feature 😄

Reply:
I realized! I was just working on this now ;)